lib/data/model/entities_const.dart

```diff
@@ -0,0 +1,31 @@
+// General
+const String CONTENT = "content";
```

Comment:
what's the thinking behind the change in approach to not use type inference?

Reply:
Perfect, now it's done but it still needing the refactor that you proposed in your last comment.